tests/slo/run.sh

```diff
@@ -0,0 +1,31 @@
+export GOOS=darwin
```

Comment:
why run script run macos arm version instead on linux x86 (or default)?

Reply:
For testing purpose, I will remove that.